docs/team/kevinchangjk.md

```diff
@@ -3,12 +3,121 @@ layout: page
+  - [#43] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/43)
+  - [#49] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/49)
+  - [#50] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/50)
+  - [#51] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/51)
+  - [#54] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/54)
+  - [#57] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/57)
+  - [#63] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/63)
+  - [#64] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/64)
+  - [#66] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/66)
+  - [#67] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/67)
+  - [#69] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/69)
+  - [#71] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/71)
+  - [#74] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/74)
+  - [#82] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/82)
+  - [#83] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/83)
+  - [#87] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/87)
+  - [#91] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/91)
+  - [#92] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/92)
+  - [#109] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/109)
+  - [#110] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/110)
+  - [#111] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/111)
+  - [#112] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/112)
+  - [#113] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/113)
+  - [#114] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/114)
+  - [#115] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/115)
+  - [#118] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/118)
+  - [#119] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/119)
+  - [#120] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/120)
+  - [#121] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/121)
+  - [#122] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/122)
+  - [#123] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/123)
+  - [#126] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/126)
+  - [#130] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/130)
+  - [#133] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/133)
+  - [#152] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/152)
+  - [#221] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/221)
+  - [#223] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/223)
+  - [#227] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/227)
+  - [#230] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/230)
+  - [#231] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/231)
+  - [#235] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/235)
+  - [#236] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/236)
+  - [#237] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/237)
+  - [#239] (https://github.com/AY2223S1-CS2103T-T15-1/tp/pull/239)
```

Comment:
Consider removing the spaces for a nice view

Reply:
Yes sir.

Reply:
Perhaps can use comma instead of a list view.